core/geometry/src/geometry3d/Point2dVector2d.ts

```diff
@@ -139,142 +196,168 @@ export class Point2d extends XY implements BeJSONFunctions {
     return Point2d.create(
       Geometry.interpolate(this.x, fractionX, other.x),
       Geometry.interpolate(this.y, fractionY, other.y),
-      result);
+      result
+    );
   }
-
   /** Return this point minus vector */
   public minus(vector: XAndY, result?: Point2d): Point2d {
     return Point2d.create(
       this.x - vector.x,
       this.y - vector.y,
-      result);
+      result
+    );
   }
-
   /** Return point plus vector */
   public plus(vector: XAndY, result?: Point2d): Point2d {
     return Point2d.create(
       this.x + vector.x,
       this.y + vector.y,
-      result);
+      result
+    );
   }
   /** Return point plus vector */
   public plusXY(dx: number = 0, dy: number = 0, result?: Point2d): Point2d {
     return Point2d.create(
       this.x + dx,
-      this.y + dy, result);
+      this.y + dy, result
+    );
   }
-
   /** Return point + vector * scalar */
   public plusScaled(vector: XAndY, scaleFactor: number, result?: Point2d): Point2d {
     return Point2d.create(
       this.x + vector.x * scaleFactor,
       this.y + vector.y * scaleFactor,
-      result);
+      result
+    );
   }
-
   /** Return point + vectorA * scalarA + vectorB * scalarB */
   public plus2Scaled(vectorA: XAndY, scalarA: number, vectorB: XAndY, scalarB: number, result?: Point2d): Point2d {
     return Point2d.create(
       this.x + vectorA.x * scalarA + vectorB.x * scalarB,
       this.y + vectorA.y * scalarA + vectorB.y * scalarB,
-      result);
+      result
+    );
   }
-
   /** Return point + vectorA * scalarA + vectorB * scalarB + vectorC * scalarC */
-  public plus3Scaled(vectorA: XAndY, scalarA: number, vectorB: XAndY, scalarB: number, vectorC: XAndY, scalarC: number, result?: Point2d): Point2d {
+  public plus3Scaled(vectorA: XAndY, scalarA: number, vectorB: XAndY, scalarB: number,
+    vectorC: XAndY, scalarC: number, result?: Point2d): Point2d {
     return Point2d.create(
       this.x + vectorA.x * scalarA + vectorB.x * scalarB + vectorC.x * scalarC,
       this.y + vectorA.y * scalarA + vectorB.y * scalarB + vectorC.y * scalarC,
-      result);
+      result
+    );
   }
   /**
    * Return the dot product of vector from this to targetA and vector from this to targetB
    * @param targetA target of first vector
    * @param targetB target of second vector
    */
   public dotVectorsToTargets(targetA: XAndY, targetB: XAndY): number {
-    return (targetA.x - this.x) * (targetB.x - this.x) +
-      (targetA.y - this.y) * (targetB.y - this.y);
+    return (targetA.x - this.x) * (targetB.x - this.x) + (targetA.y - this.y) * (targetB.y - this.y);
   }
-
-  /** Returns the (scalar) cross product of two points/vectors, computed from origin to target1 and target2 */
+  /**
+   * Returns the (scalar) cross product of vector from this to targetA and vector from this to targetB
+   * @param targetA target of first vector
+   * @param targetB target of second vector
```

Comment:
targetA and targetB don't agree with the names of the arguments

Reply:
will fix